Web/starter_project_vanguard/components/layout/links/socialMediaLinks.tsx

```diff
@@ -0,0 +1,31 @@
+import { FaFacebook, FaInstagram, FaLinkedin,FaTwitter,FaYoutube} from 'react-icons/fa'
+
+interface iconLink {
```

Comment:
[nit] type names should be in PascalCase (but actually you can also leave these type definitions out if you don't want them - it is not a bad thing to have them tho)